src/modules/mavlink/mavlink_params.c

```diff
@@ -166,3 +166,15 @@ PARAM_DEFINE_INT32(MAV_ODOM_LP, 0);
  * @max 250
  */
 PARAM_DEFINE_INT32(MAV_RADIO_TOUT, 5);
+
+/**
+ * Message forwarding for USB MAVLink instance.
+ *
+ * When set, messages are forwarded to the USB MAVLink instance.
+ * This is equivalent to e.g. the MAV_0_FORWARD, for MAVLink instance 0.
+ *
+ * @boolean
+ * @reboot_required true
+ * @group MAVLink
+ */
+PARAM_DEFINE_INT32(MAV_S_FORWARD, 0);
```

Comment:
FMI 
- Why "S"? I.e. not very intuitive naming.
- Does it make sense to treat USB as just another serial port you can assign a mavlink instance to - ie. such that you could assign MAV_1_FORWARD to it? My concern is discoverability.
- Is there only ever one USB interface/mavlink instance? What if there are two?

I don't really need an answer - these might be stupid questions. Just what came to my head when I saw this.